_posts/one-password.md

```diff
@@ -0,0 +1,75 @@
+
+## What's next?
+
+I planned for a future switch when I upgraded my account to premium. I set a reminder for me nine months ahead to start looking for alternatives with a significant enough buffer time before the LastPass renewal.
+
+When the time came, based on the early cost of a password management solution, I decided not to go through the pain of other password manager trials. I didn't want to update all my devices to new software with a trial license. Import all my passwords, play with the bits, and switch to a different trial.
+
+Based on friends' and colleagues' recommendations, I picked [1Password](https://1password.com) and went for a 1-year Personal subscription. I only checked two things:
+
+- The features listed on the website satisfied my requirements
+- There was support to import from LastPass
+
+### So what?
+
+I've been using 1Password for six months, and the balance is not that great.
+
+Is it better than LastPass? Yes and no. It depends.
+
+The biggest issue is GDPR related. By default, data are stored outside the European Union, posing a privacy concern. There is an option, at subscription time, to store data in the EU; however, the sign-up process doesn't highlight it enough (it's a euphemism). Afterward, it's possible to move data across regions, but the process is manual, cumbersome, and far from being user-friendly, which is never a good sign, in my opinion. I'll try it sooner or later.
+
+The second issue is 2FA-related. 1Password, as LastPass does, supports storing the two-factor authentication token in the password manager. It's a handy feature, nonetheless. However, my spider sense perceives it as a security risk. There are scenarios in which it might be a [good enough compromise](https://security.stackexchange.com/questions/194142/is-it-safe-to-store-2fa-tokens-together-with-passwords-in-1password#194279), and [others](https://polansky.co/blog/psa-2fa-password-managers/). So I decided to continue using [Authy](https://authy.com) as the 2FA token storage.
+
+And there comes the third issue. It seems impossible, or at least I couldn't find a way, to disable 2FA support in 1Password. That means 1Password will try to fill tokens when requested automatically. It's again a user experience problem and isn't very pleasant.
+
+The last issue I faced so far is again UX related. On desktop computers, either macOS or Windows, the browser extension comes with a companion application that runs on the operating system. As far as I have understood, it's the companion application that stores data and takes care of encryption. It is probably a more secure solution than doing all that in the browser extension, as LastPass does. However, it comes with some UX issues. For example, the most annoying one happens when clicking on the browser extension, and a password is required to unlock 1Password. Sometimes, after inputting the password, the companion application pops up, full screen, in the foreground.
```

Comment:
I've had some UX annoyances with v8 as well, especially when I first start my machine, but I've never had to click on the browser extension. Instead, I just press the down arrow in a field and 1Password just shows up with the saved logins for that page.

Some other possible mitigations:

- Use the keyboard shortcut to launch 1Password whenever you know you're going to a secure site. E.g. if I know I need to do my banking, I press Cmd-\ to bring up the 1Password pop-up thing, select my password and press option-Enter to open the website and automatically fill it in
- Set 1Password to save your password for the maximum time (2 weeks, I think?) so you don't have to enter it each time. Depends how secure your device is, of course
- Consider using something like Alfred which has a 1Password workflow. I prefer LaunchBar which doesn't have native integration with v8 so I have to stick with the first item in this list.

One nice new thing with 1Password v8 is that you can use it to fill in passwords for native apps now, not just in the browser. At least on Mac, not sure if that works on Windows. Useful for things like NordVPN and Steam. The actual autofill part is spotty at best but you can still use the Cmd-\ shortcut and then Shift-Cmd-C to copy the password in those cases.

Reply:
☝️ might be better suited as a comment on the actual post. Let me know if you'd rather I posted it there

Reply:
Yes, please!